src/planscape/stands/services.py

```diff
@@ -112,7 +112,7 @@ def calculate_stand_vector_stats3(
     WITH centroid AS (
         SELECT
             id,
-            ST_Centroid(geometry) as "geometry"
+            ST_Centroid(geometry) as "geometry",
```

Comment:
```suggestion
            ST_Centroid(geometry) as "geometry"
```
Comma can break the query.